contracts/interfaces/IERC7984Rwa.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import {FHE, externalEuint64, euint64} from "@fhevm/solidity/lib/FHE.sol";
+import {IAccessControl} from "@openzeppelin/contracts/access/IAccessControl.sol";
+import {IERC165} from "@openzeppelin/contracts/interfaces/IERC165.sol";
+import {IERC7984} from "./IERC7984.sol";
+
+/// @dev Base interface for confidential RWA contracts.
+interface IERC7984RwaBase {
+    /// @dev Emitted when the contract is paused.
+    event Paused(address account);
+    /// @dev Emitted when the contract is unpaused.
+    event Unpaused(address account);
+
+    /// @dev The caller account is not authorized to perform an operation.
+    error OwnableUnauthorizedAccount(address account);
+    /// @dev The owner is not a valid owner account. (eg. `address(0)`)
+    error OwnableInvalidOwner(address owner);
+    /// @dev The operation failed because the contract is paused.
+    error EnforcedPause();
+    /// @dev The operation failed because the contract is not paused.
+    error ExpectedPause();
```

Comment:
Are these errors utilized? What's the point in having them here?

Reply:
I removed `ExpectedPause` which was unused.
`ExpectedPause` is eventually fired by `whenNotPaused`, I think devs should be able to fetch that error by relying on interface instead of implem, don't you you think so?

Reply:
I don't think its crucial to define errors in the interface... most of our interfaces don't do this. We shouldn't define errors that aren't utilized imo.

Reply:
Yes, removed in [Expose single rwa interface](https://github.com/OpenZeppelin/openzeppelin-confidential-contracts/pull/160/commits/e94961e31156352dd0637489e3e03117232b73d8).